feature-utils/poly-look/src/react-components/inputs/common.css

```diff
@@ -0,0 +1,82 @@
+.poly-input-field {
+  display: flex;
+  flex-direction: column;  
+}
+
+.poly-input-field .body {
+  height: 56px;
+  border: 1px solid var(--pl-field-color-border-enabled);
+  padding: var(--pl-field-spacing-padding-hor)
+    var(--pl-field-spacing-padding-vert);
+  border-radius: var(--pl-radius-1x);
+  display: flex;
+  align-items: center;
+  position: relative;
+  box-sizing: border-box;
+  background-color: var(--pl-field-color-fill-enabled);
+  font-size: var(--font-size-ml);
+}
+
+.poly-input-field .body-focused {
+  background-color: var(--pl-field-color-fill-focused);
+  border-color: var(--pl-field-color-border-focused);
+}
+
+.poly-input-field .body-error {
+  background-color: var(--pl-field-color-fill-error);
+  border-color: var(--pl-field-color-border-error);
+}
+
+.poly-input-field .body-error+.helper-text {
+  color: var(--pl-field-color-text-error);
+}
+
+.poly-input-field .body-disabled {
+  pointer-events: none;
+  background-color: var(--pl-field-color-fill-disabled);
+  border-color: var(--pl-field-color-border-disabled);
+  color: var(--pl-field-color-text-disabled);
+}
+
+.poly-input-field .body-disabled+.helper-text {
+  color: var(--pl-field-color-text-disabled);
+}
+
+.poly-input-field input {
+  flex: 1;
+  outline: none;
+  border: none;
+  padding: 0;
+  font: inherit;
+  line-height: inherit;
+  letter-spacing: inherit;
+  background-color: inherit;
+  margin-right: var(--pl-field-spacing-gap);
+}
+
+.poly-input-field label {
+  position: absolute;
+  font: inherit;
+  line-height: inherit;
+  letter-spacing: inherit;
+  left: var(--pl-field-spacing-padding-vert);
+  top: 50%;
+  transform: translateY(-50%);
+  transition: transform 0.2s, font-size 0.2s;
+}
+
+.poly-input-field .label-up {
+  font-size: var(--font-size-s);
+  transform: translateY(-125%);
```

Comment:
I was testing it out in the browser, and though it's true in Figma I have no spacing between these 2 elements, the lineheight it their own spacing. I was 150% worked well 

<img width="1207" alt="Screenshot 2022-10-14 at 14 24 31" src="https://user-images.githubusercontent.com/7642309/195846237-f14e67a3-d252-40c1-ba72-e74f220d6850.png">

Reply:
And the position of the label text is not correct, the label and the input text should be vertically centered in the Container
<img width="657" alt="Screenshot 2022-10-14 at 13 48 12" src="https://user-images.githubusercontent.com/7642309/195846547-3353ccd2-8467-40f1-9854-db2600d5489a.png">

Here is a video of the interaction and position of the elements
https://firebasestorage.googleapis.com/v0/b/design-spec/o/projects%2Fm3%2Fimages%2Fl13ubrf0-Label%20text_3P.mp4?alt=media&token=37ec7f1f-0921-4d77-a87c-1e8ff247e779

Reply:
Getting the position exactly as in Figma might be doable, but I couldn't do it. It's mostly guess work here, because of that position absolute on the label. I've made some changes and added a negative margin on the input, maybe they are better now.

Reply:
I see an issue with line-heights here, coming from our stylings in Figma, line-heights should also follow a 4px increment, but now they dont. Also the borders reduce the box element to 54px which doesn't help calculations.

Ideally 
- Label should have a line height of 20px and translated -20px (when in prefilled mode)
- Input should have a line height of 20px and sitting on top of the bottom padding.
- For that we could update `--pl-field-spacing-padding-vert` to `var(--pl-space-2x)`

Would this work for you?

Reply:
Done